README.md

```diff
@@ -28,14 +28,13 @@ $ npx sass ./test/main.scss --load-path=./bower_components/
 
 ### Developing with a next app
 
-Currently the best way to develop with your changes on the site is to use [`bower link`](https://bower.io/docs/api/#link) in a running app.
+~~Currently the best way to develop with your changes on the site is to use [`bower link`](https://bower.io/docs/api/#link) in a running app.~~ Unfortunately there more instructions needed to develop this library within a next app. There is ticket [ACC-1156](https://financialtimes.atlassian.net/browse/ACC-1156) to investigate the best way to develop this library within a next app.
```

Comment:
[Missing word] "there more" looks like it's missing "are"

Reply:
Is there a reason you're leaving the "~~Currently the best way to develop with your changes on the site is to use [`bower link`](https://bower.io/docs/api/#link) in a running app." sentence? Not sure it adds anything, though could be added for context to the ACC-1156 ticket